test/powershell/Modules/Microsoft.PowerShell.Utility/Get-Date.Tests.ps1

```diff
@@ -167,3 +168,105 @@ Describe "Get-Date" -Tags "CI" {
         $timeDifference.Ticks        | Should -BeLessThan 10000
     }
 }
+
+Describe "Get-Date -UFormat tests" -Tags "CI" {
+    BeforeAll {
+        $date1 = Get-date -Date "2030-4-5 1:2:3.09"
```

Comment:
Is there a reason these specific date/times were chosen?  I recall from the week of the year (ISO 8601) thread there were some edge cases like the first week starting on a certain day of the week or the end of the year being on a certain day of the week so that.

Reply:
For ISO8601 we have separate tests.
Here I pick up different numers for every position to make it easier to see them as a result.
$date2 and $date3 is for testing padding and time formats 1-12 or 0-23

Reply:
Got it.  Thanks